include/tvm/relay/expr.h

```diff
@@ -240,14 +245,16 @@ class Var : public Expr {
  * \param opt_vid The (optional) vid for the copied var. If none, ret_var->vid = var->vid.
  * \param opt_type_annotation The (optional) type_annotation for the copied var. If none,
  * ret_var->type_annotation = var->type_annotation.
- * \param opt_span The (optional) span for the copied var. If none, ret_var->span = var->span.
- * \return If all properties are null or the same as the property in the input var
- * (i.e., opt_vid is null or opt_vid.value() == var->vid, etc.), then we return var. Otherwise,
- * we return a copy of call with the different fields overwritten. (i.e., if
- * opt_vid.value() != var->vid, then ret_var->vid = opt_.value()).
+ * \param opt_virtual_device The (optional) virtual_device for the copied tuple. If none,
+ * ret_tuple->virtual_device = tuple->virtual_device. \param opt_span The (optional) span for the
+ * copied var. If none, ret_var->span = var->span. \return If all properties are null or the same as
+ * the property in the input var (i.e., opt_vid is null or opt_vid.value() == var->vid, etc.), then
+ * we return var. Otherwise, we return a copy of call with the different fields overwritten. (i.e.,
+ * if opt_vid.value() != var->vid, then ret_var->vid = opt_.value()).
```

Comment:
\param and \return need new lines.

Reply:
Thanks! for some reason clang-format kept squishing them into one line.